### PR TITLE
Give version to installWarpAPI

### PR DESCRIPTION
**Pull request by yohannroussel**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "warp",
-    "version": "1.0.0-eng.0",
+    "version": "1.1.0-eng.0",
```
Comment (jpl-scale) [inline package.json]: This is not a public api. I think we can stick to something like `1.0.1`

Comment (jpl-scale) [reply package.json]: And, if we do, there is no need to increment the PL

Comment (jpl-scale) [reply package.json]: In fact, this is not part of the warp API, but part of the runtime API. So, definitely no change here.

File: warp.js [hunk 2]
```diff
@@ -24,5 +24,5 @@ module.exports = {
 };
 
 if (global.warp) {
-    global.warp.runtime.installWarpAPI(module.exports);
+    global.warp.runtime.installWarpAPI(module.exports, require('./package.json').version);
```
Comment (jpl-scale) [inline warp.js]: So, may be we need to swap parameter?

File: package.json (hunk 1 above)
Comment (yohannroussel) [reply package.json]: This change doesn't impact compatibility with users but it changes compatibility with runtimes. I'm not confortable with keeping the version unchanged.

Comment (jpl-scale) [reply package.json]: To be clear, we have no need to increment the PL because the CI manage that for you.

**Pull request merged**